src/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Categories = () => {
+	const CategoriesContainer = styled.div`
+	color: #ffffff;
+	font-weight:bold;
```

Comment:
Again - formatter. You should use it on save: https://www.educative.io/answers/how-to-set-up-prettier-and-automatic-formatting-on-vs-code